svf/include/SVFIR/SVFIR.h

```diff
@@ -578,6 +578,13 @@ class SVFIR : public IRGraph
         return addValNode(nullptr, node, i);
     }
 
+    NodeID addArgValNode(const CallGraphNode* callGraphNode, u32_t argNo,
+                         bool isUncalled, NodeID i, const ICFGNode* icfgNode) {
```

Comment:
put ` bool isUncalled` as the last parameter and a default parameter to be false.